clustered/server/build.gradle

```diff
@@ -29,6 +29,27 @@ dependencies {
   provided "org.terracotta:standard-cluster-services:$parent.terracottaApisVersion"
 }
 
+compileJava {
+  options.fork = true;
+  options.forkOptions.executable = MavenToolchain.javaExecutable(JavaVersion.VERSION_1_8, 'javac')
+}
+
+compileTestJava {
+  options.fork = true;
+  options.forkOptions.executable = MavenToolchain.javaExecutable(JavaVersion.VERSION_1_8, 'javac')
+}
+
+javadoc {
+  options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
This will fail javadoc build when running with a JDK different than 1.8.
Either make it conditional or add the `executable` attribute to mandate usage of 1.8